zuul.d/jobs.yaml

```diff
@@ -755,18 +755,51 @@
       - playbooks/ansible-test-base/post.yaml
     required-projects:
       - name: github.com/ansible/ansible
+      - name: github.com/ansible-collections/openvswitch.openvswitch
     timeout: 3600
     vars:
+      ansible_collections_repo: github.com/ansible-collections/openvswitch.openvswitch
       ansible_test_command: network-integration
       ansible_test_integration_targets: "openvswitch_.*"
 
+- job:
+    name: ansible-test-network-unit-openvswitch
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    parent: ansible-network-openvswitch-appliance
+    pre-run:
+      - playbooks/ansible-test-base/pre.yaml
+      - playbooks/ansible-test-network-integration-base/pre.yaml
```

Comment:
We shouldn't need to step into the ansible-test-network-integration-base playbooks, we can remove this